odoo/tools/translate.py

```diff
@@ -443,8 +443,16 @@ def _get_lang(self, frame):
                     lang = env['res.users'].context_get()['lang']
         return lang
 
-    def __call__(self, source):
-        return self._get_translation(source)
+    def __call__(self, source, *args, **kw):
+        translation = self._get_translation(source)
+        if args or kw:
```

Comment:
Maybe add an error if `args` *and* `kw`? Especially given `kw` gets ignored entirely in that case.